src/solvers/dgmulti/flux_differencing_gauss_sbp.jl

```diff
@@ -205,6 +218,12 @@ end
   # for 2D GaussSBP nodes, the indexing is first in x, then y
   out = reshape(out_vec, nnodes_1d, nnodes_1d)
 
+  if ApplyFaceWeights == Static.True
```

Comment:
```suggestion
  if ApplyFaceWeights == true # allows to use `Bool`s and `Static.StaticBool`s
```
If you switch to instances, this will work - and also work with `Bool`s